app/connectors/propertyLinking/PropertyLinkConnector.scala

```diff
@@ -103,29 +103,27 @@ class PropertyLinkConnector @Inject()(config: ServicesConfig, http: HttpClient)(
         pagination: PaginationParams,
         organisationId: Long,
         agentOrganisationId: Long,
-        agentAppointed: Option[String] = None
+        agentAppointed: Option[String] = None,
+        agentCode: Long
   )(implicit hc: HeaderCarrier): Future[OwnerAuthResult] = {
-
-    val ownerAuthResult = http.GET[OwnerAuthResult](
-      s"$baseUrl/owner/property-links/appointable",
-      List(
-        searchParams.address.map("address"  -> _),
-        searchParams.baref.map("baref"      -> _),
-        searchParams.agent.map("agent"      -> _),
-        searchParams.status.map("status"    -> _),
-        Some("sortfield"                    -> searchParams.sortfield.toString),
-        Some("sortorder"                    -> searchParams.sortorder.toString),
-        agentAppointed.map("agentAppointed" -> _),
-        Some("organisationId"               -> organisationId.toString),
-        Some("agentOrganisationId"          -> agentOrganisationId.toString)
-      ).flatten ++
-        List(
-          "startPoint"           -> pagination.startPoint.toString,
-          "pageSize"             -> pagination.pageSize.toString,
-          "requestTotalRowCount" -> pagination.requestTotalRowCount.toString
+    val ownerAuthResult = agentAppointed.getOrElse("BOTH") match {
```

Comment:
"BOTH" could be replaced by "foobar" and it would still work. I think in this case an if-statement would be better. 

```scala
if (agentAppointed.contains("NO") { getMyOrganisationsPropertyLinks(searchParams, pagination) } else {...}
```